tests/wpunit/Tribe/Events/ORM/Venues/FetchTest.php

```diff
@@ -81,10 +81,9 @@ public function should_allow_getting_venues_by_phone() {
 	 *
 	 * @test
 	 */
-	public function should_allow_getting_venues_by_postal_code() {
+	public function jhould_allow_getting_venues_by_postal_code() {
```

Comment:
`jhould`?

Reply:
It's Danish for "it should"; I'm surprised you don't know.